activesupport/test/cache/behaviors/cache_logging_behavior.rb

```diff
@@ -56,18 +56,27 @@ def test_exist_logging
   def test_read_multi_logging
     assert_logs("Cache read_multi: 1 key(s)") { @cache.read_multi("foo") }
     assert_logs("Cache read_multi: 2 key(s)") { @cache.read_multi("foo", "bar") }
+
+    assert_logs("Cache read_multi: 1 key(s)") { @cache.read_multi("foo", namespace: "bar") }
+    assert_logs("Cache read_multi: 2 key(s)") { @cache.read_multi("foo", "bar", namespace: "bar") }
```

Comment:
Specifying a static namespace could cause tests to interfere with each other in the same way as having no namespace.

We could instead generate a random namespace for these assertions, but I'm not sure how much that buys us because (1) the expected value for these assertions is the same with or without a namespace, and (2) the Redis and Memcache tests are implicitly testing this code with a namespace.

I am more inclined to simply omit these assertions.  Even more so since they are not related to the fix.

Reply:
I see what you mean and will consider an alternative - my change is maybe more applicable for #50200 which involves read_multi, but still needs more thought re: use of namespace

Reply:
omitting as suggested